packages/react-composites/src/composites/CallComposite/hooks/useHandlers.ts

```diff
@@ -150,7 +150,7 @@ const createCompositeHandlers = memoizeOne(
       },
       /* @conditional-compile-remove(call-readiness) */
       askDevicePermission: async (constrain) => {
-        return adapter.askDevicePermission(constrain);
+        await adapter.askDevicePermission(constrain);
```

Comment:
do we not need to return the await here?

Reply:
Is a good question, we don't only because this is a handler for a UI component that doesn't care about the result

So the UI component's interface returns void:
```
<PermissionsModal askDevicePermissions={async () => void} />
```